settings.adoc

```diff
@@ -1,9 +1,9 @@
-:document-title: Administrator Guide
+:document-title: Administrator Guide version 8.8.5
```

Comment:
nit: are we able to take advantage of {product-version} here (perhaps by rearranging order?) to avoid the need of modifying the version number in two places?